spec/formtastic_integration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# rubocop:disable AlignArray
+require 'spec_helper'
+require 'formtastic'
+require 'formtastic/inputs/trix_editor_input'
+
+describe Trix do
+  include ActionPack
```

Comment:
Is there a simpler way to require the global state of all of these modules, without manually including each one?